app/components/shared/forms/Input.ts

```diff
@@ -349,30 +356,41 @@ export function getPropertiesFormData(obsSource: obs.ISource): TFormData {
 
     if (isNumberProperty(obsProp)) {
       if (obsProp.details.type === obs.ENumberType.Slider) {
-        Object.assign(formItem as ISliderInputValue, {
+        (formItem as INumberInputValue) = {
```

Comment:
Hmm, it looks like you're re-assigning a `const` declaration, which should result in an exception.  Is this working as expected?  Maybe I'm missing something.

Reply:
Herm good point. Seems making formItem non-const also has typescript complain that it's never re-assigned though which is odd.

Reply:
And yes, it appears to work correctly.